delivery_package_number/__manifest__.py

```diff
@@ -10,14 +10,13 @@
     "license": "AGPL-3",
     "installable": True,
     "application": False,
-    "depends": ["delivery"],
+    "depends": ["delivery", "stock_delivery"],
```

Comment:
Why this new dependency?

Reply:
it's a new glue module for pickings + delivery, the base `delivery` only handles the carrier models and adds shipping methods to sales now. Though the depends could be abbreviated to just `["stock_delivery"]`

Reply:
OK, I see. Maybe we should rename this to `stock_delivery_package_number`? cc @chienandalu @sergio-teruel

Reply:
Odoo hasn't renamed any of their modules to `stock_delivery_x` so far, not even another glue module `delivery_stock_picking_batch`. I think it would more consistent to keep the current `delivery_x` naming